docs/gpu.md

```diff
@@ -74,3 +74,34 @@ Epoch 1 train begin 06:12:38
 ```
 ## AMP (AUTOMATIC MIXED PRECISION)
 AMP is very useful on GPU training and PyTorch/XLA reuse Cuda's AMP rule. You can checkout our [mnist example](https://github.com/pytorch/xla/blob/master/test/test_train_mp_mnist_amp.py) and [imagenet example](https://github.com/pytorch/xla/blob/master/test/test_train_mp_imagenet_amp.py). Note that we also used a modified version of [optimizers](https://github.com/pytorch/xla/tree/master/torch_xla/amp/syncfree) to avoid the additional sync between device and host.
+
+## Develop PyTorch/XLA on a GPU instance (build PyTorch/XLA from source with GPU support)
```

Comment:
can you add a refernce in https://github.com/pytorch/xla/blob/master/CONTRIBUTING.md#building-manually to this doc?

Reply:
done